docs/examples/table/controlledExpandable.js

```diff
@@ -0,0 +1,73 @@
+// @flow strict
+import { useState, type Node } from 'react';
+import { Table, Flex, Text, Box } from 'gestalt';
+
+export default function Example(): Node {
+  const [expandedRows, setExpandedRows] = useState(['row1']);
+
+  return (
+    <Box height="100%" width="100%" overflow="scroll">
+      <Flex alignItems="start" gap={4} height="100%" justifyContent="center" width="100%">
+        <Table accessibilityLabel="Table Row Expandable">
+          <Table.Header>
+            <Table.Row>
+              <Table.HeaderCell>
+                <Box display="visuallyHidden">
+                  <Text weight="bold">Open/Close row</Text>
+                </Box>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Campaign</Text>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Status</Text>
+              </Table.HeaderCell>
+              <Table.HeaderCell>
+                <Text weight="bold">Date</Text>
+              </Table.HeaderCell>
+            </Table.Row>
+          </Table.Header>
+
+          <Table.Body>
+            {[
+              { id: 'row1', campaign: 'Winter', status: 'Draft', date: 'December 25, 2023' },
+              { id: 'row2', campaign: 'Spring', status: 'Active', date: 'April 25, 2023' },
+              { id: 'row3', campaign: 'Fall', status: 'Finished', date: 'October 25, 2022' },
+            ].map((x) => (
```

Comment:
`x` isn't a great variable name. Since the variable clearly isn't important here, consider destructuring the needed properties inline to skirt the issue completely:
```js
].map(({ id, campaign, status, date }) => (
```

Reply:
DONE